api/controllers/FlairController.js

```diff
@@ -178,11 +178,11 @@ module.exports = {
         var ipAddress = req.headers['x-forwarded-for'] || req.ip;
         Event.create([{
           type: "flairTextChange",
-          user: req.user.name,
+          userName: req.user.name,
```

Comment:
Either of them would work, but at the moment the search function displays the userName property. Changing it to user would require us to migrate all of the existing events.